cockroachdb/README.md

```diff
@@ -73,6 +73,25 @@ role with given bucket access. Role with bucket access should be configured in t
 
 AWS credentials are injected via [vault](https://github.com/utilitywarehouse/documentation/blob/master/infra/vault/vault-aws.md)
 
+## Init Jobs
+
+These manifests provide two Jobs to initialise parts of the DB:
+
+  - Configure the database: This expects the environment variables `DB_USER` and
```

Comment:
I think we should not add this information to README. It already lives in the example directory. 
Why would we keep it in two places at once?

Reply:
> I intended to have all the configuration information there, and none in README

:+1: I missed this. Does that include the job deletion note I added here? Where could that live, maybe just as a comment in the manifests?

Reply:
>  Where could that live, maybe just as a comment in the manifests?

I think that's a good idea, I would go with it :) Higher chance somebody interested in this particular job will spot it there than in README

Reply:
> > Where could that live, maybe just as a comment in the manifests?
> 
> I think that's a good idea, I would go with it :) Higher chance somebody interested in this particular job will spot it there than in README

moving docs into example+manifests: [00d1d07](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/58/commits/00d1d074bf8b04043b3e878096167c57dad83382)